src/ToolBox/superpmi/superpmi-shim-collector/icorjitcompiler.cpp

```diff
@@ -63,9 +63,6 @@ CorJitResult __stdcall interceptor_ICJC::compileMethod(ICorJitInfo*
         our_ICorJitInfo.mc->recGlobalContext(*g_globalContext);
     }
 
-    // Record a simple view of the environment
-    our_ICorJitInfo.mc->recEnvironment();
```

Comment:
This call dumped environment variables during the collection, how was it replaced?

Reply:
It wasn't. We could certainly add the capability to dump the JIT config settings during/after a collection, though.

Reply:
During SuperPMI collection some tests set flags as complus_jitstress, complus_enableavx and etc. SpmiShim record them and the spmi read them from mc file and apply for the correct replay. It is critical to keep this mechanism working.

Reply:
Ah, I misunderstood your question. That mechanism will continue to work: those settings will be captured by the JitHost.

Reply:
Cool, now I see it.